### PR TITLE
[Refactor] HIVE_PARTITION_STATS_SAMPLE_SIZE const string error

### PR DESCRIPTION
**Pull request by xiaoyong-z**
Signed-off-by: xyz <a997647204@gmail.com>

## What type of PR is this：
- [ ] BugFix
- [ ] Feature
- [ ] Enhancement
- [x] Refactor
- [ ] UT
- [ ] Doc
- [ ] Tool

## Which issues of this PR fixes ：

Fixes #

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] I have added user document for my new feature or new function

### CODE REVIEW

**Comment by xiaoyong-z** [comment]
run starrocks_fe_unittest

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

**Review by stephen-shelby — APPROVED** [review]
LGTM

File: fe/fe-core/src/main/java/com/starrocks/qe/SessionVariable.java [hunk 1]
```diff
@@ -258,7 +258,7 @@ public class SessionVariable implements Serializable, Writable, Cloneable {
     // In most cases, the partition statistics obtained from the hive metastore are empty.
     // Because we get partition statistics asynchronously for the first query of a table or partition,
     // if the gc of any service is caused, you can set the value to 100 for testing.
-    public static final String HIVE_PARTITION_STATS_SAMPLE_SIZE = "3000";
+    public static final String HIVE_PARTITION_STATS_SAMPLE_SIZE = "hive_partition_stats_sample_size";
```
Comment (chaoyli) [inline fe/fe-core/src/main/java/com/starrocks/qe/SessionVariable.java:261]: What's the problem with the const variable?

**Review by chaoyli — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
run starrocks_admit_test

**Review by meegoo — APPROVED** [review]

**Comment by xiaoyong-z** [comment]
run starrocks_admit_test

**Pull request merged**